content/en/tracing/troubleshooting/iis_dotnet_framework.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Troubleshooting .NET Framework Traces
+kind: documentation
+---
+
+## Problem 
+
+.NET Framework APM traces are not showing within Datadog.
+
+## Solution
+
+If .NET Framework APM traces are not showing within Datadog, review the possible reasons below.
+
+### Compatability 
+
+The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and above.   Additionally, only [supported frameworks][3] are automatically instrumented.  If you are are missing spans from a part of your application, please confirm the integration and version are supported and [open a support ticket][4] with this information provided to expedite our review.
+
+
+### Confirm installation 
+
+#### Datadog Agent
+
+Verify the Datadog Agent for Windows OS been installed locally. If not, visit the [Datadog Windows Agent][1] guide.
+
+#### Datadog .NET tracer
+
+Verify the Datadog .NET tracer been installed locally. If not, visit our [Datadog .NET APM release page][2].
+
+### Automatic instrumentation 
+
+#### Environment variables
+
+For automatic instrumentation of IIS applications, the required environment variables must be set within the `W3SVC` & `WAS` Windows registry paths. 
+
+##### Steps
+
+1. Run `regedit.exe` to open the Windows Registry Editor.
+2. Check the following paths: 
+```text
+HKLM:SYSTEM\CurrentControlSet\Services\W3SVC
+HKLM:SYSTEM\CurrentControlSet\Services\WAS
+```
+3. Create a multi-string value: 
```

Comment:
"editing these specific keys by hand has a big chance of user error that will break the tracer"

- Please take a look at the staging link. https://docs-staging.datadoghq.com/jay.dave/dotnet_troubleshooting/tracing/troubleshooting/iis_dotnet_framework
- We are giving them a copy feature so they do not type anything incorrectly. 

"instead of doing this, users should reinstall or repair the msi installer, which is easier and safer"
- We should be offering alternative ways for setting this outside of saying to the customer if there is an issue uninstall and reinstall or repair, which is why the PS is being shared.

Reply:
Can we take this discussion back to the design document?